ubuntu24.04/Dockerfile

```diff
@@ -67,9 +67,7 @@ ADD install.sh /tmp
 RUN apt-key del 7fa2af80 && OS_ARCH=${TARGETARCH/amd64/x86_64} && OS_ARCH=${OS_ARCH/arm64/sbsa} && \
     apt-key adv --fetch-keys "https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2404/${OS_ARCH}/3bf863cc.pub"
 
-RUN /tmp/install.sh reposetup && /tmp/install.sh depinstall && \
```

Comment:
Question -- are we able to remove line 91-92 now?
```
ARG PUBLIC_KEY=empty
COPY ${PUBLIC_KEY} kernel/pubkey.x509
```

Reply:
Yes, we can remove this